docs/sdk/server-side-sdks/dotnet-local/dotnet-local-gettingstarted.md

```diff
@@ -24,10 +24,52 @@ namespace Example
     {
         static Main(string[] args)
         {
-            DVCLocalClientBuilder apiBuilder = new DVCLocalClientBuilder();
-            using DVCLocalClient api = apiBuilder.SetSDKKey("<DVC_SERVER_SDK_KEY>")
-                      .Build();
+            DVCLocalClientBuilder clientBuilder = new DVCLocalClientBuilder();
+            using DVCLocalClient client = clientBuilder.SetSDKKey("<DVC_SERVER_SDK_KEY>").Build();
         }
     }
 }
 ```
+
+## Intialization With Callback
+
+You can also setup a callback to be notified when the client is fully initialized and use `DVCLocalOptions` to further configure the client.
+
+```csharp
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using DevCycle.SDK.Server.Local.Api;
+using DevCycle.SDK.Server.Common;
+using Microsoft.Extensions.Logging;
+
+namespace Example {
+    public class Example {
+        private static DVCLocalClient client;
+        
+        static async Task Main(string[] args) {
+            DVCLocalClientBuilder clientBuilder = new DVCLocalClientBuilder();
+            client = clientBuilder.SetSDKKey("<DVC_SERVER_SDK_KEY>")
```

Comment:
We should set a best practice of sourcing this from an env var even in our docs.